### PR TITLE
Story 4 & 5 Complete - Name/Calories throw error if empty

### PR DESCRIPTION
**Pull request by mdevoe12**
@MikelSage @case-eee 

-built functions to throw errors under input fields for empty strings
-upon successful entry, errors delete and input fields clear

### CODE REVIEW

File: lib/foods.js [hunk 1]
```diff
@@ -45,3 +39,27 @@ function foodRows(food) {
...
+      name.val("")
+      calories.val("")
+    }
+  })
+}
+
+function validateInputs(name, calories) {
+  let args = Array.from(arguments);
+  args.forEach(function(element) {
+    let message = element.attr('name').replace('-', ' ')
```
Comment (MikelSage) [inline lib/foods.js:60]: Nice use of name attributes

**Review by MikelSage — APPROVED** [review]
Nice job!

**Comment by MikelSage** [comment]
image

File: lib/foods.js [hunk 2]
```diff
@@ -9,18 +9,12 @@ $(document).ready( function() {
...
-      url: api + '/api/v1/foods',
-      data: {food: {name: name, calories: calories}},
-      success: function(data) {
-        alert('Added food!')
-        foodRows(data);
-      }
-    })
+    // let name = $.trim($('[name=food-name]').val());
+    // let calories = $.trim($('[name=calories]').val());
+    let name = $('[name=food-name]');
```
Comment (case-eee) [inline lib/foods.js:14]: if the variable is a jQuery object (which it is in this case) - then we typically name it with a $ in front of it - `$name` so it's more explicit!

File: lib/foods.js [hunk 3]
```diff
@@ -9,18 +9,12 @@ $(document).ready( function() {
...
+    let calories = $('[name=calorie-amount]');
```
Comment (case-eee) [inline lib/foods.js:15]: if we aren't using `this` or `event.target` to select these values, is there a reason not to just find them in our function calls below? do we need to pass them in?

File: lib/foods.js [hunk 4]
```diff
@@ -9,18 +9,12 @@ $(document).ready( function() {
 function setFoodListener() {
   $('#food-form').on('submit', function(event) {
```
Comment (case-eee) [inline lib/foods.js:10]: i'd love to see y'all extract out this entire callback function 👍

File: lib/foods.js [hunk 5]
```diff
@@ -45,3 +39,27 @@ function foodRows(food) {
       </tr> `
   );
 };
+
+function postFood(name, calories) {
```
Comment (case-eee) [inline lib/foods.js:43]: love that this is extracted out!

File: lib/foods.js [hunk 6]
```diff
@@ -45,3 +39,27 @@ function foodRows(food) {
...
+  $.ajax({
+    method: 'POST',
+    url: api + '/api/v1/foods',
+    data: {food: {name: $.trim(name.val()), calories: $.trim(calories.val())}},
+    success: function(data) {
```
Comment (case-eee) [inline lib/foods.js:48]: could you move this callback into a named function?!

File: lib/foods.js [hunk 7]
```diff
@@ -45,3 +39,27 @@ function foodRows(food) {
...
+      foodRows(data);
+      $('.error-message').remove()
+      name.val("")
+      calories.val("")
+    }
+  })
+}
+
+function validateInputs(name, calories) {
+  let args = Array.from(arguments);
```
Comment (case-eee) [inline lib/foods.js:58]: good use of `arguments`!

**Review by case-eee — COMMENTED** [review]
@mdevoe12 @MikelSage good work y'all 👍

**Pull request merged**